lib/kuzzle/kuzzle.ts

```diff
@@ -606,8 +607,32 @@ class Kuzzle extends KuzzleEventEmitter {
 
     try {
       for (const [type, importMethod] of Object.entries(this.importTypes)) {
+        const importPayload = {};
+
+        switch (type) {
+          case 'fixtures':
+            _.set(importPayload, ['toSupport', 'fixtures'], toSupport.fixtures);
```

Comment:
You should use the dot notation to access object elements
```suggestion
            _.set(importPayload, 'toSupport.fixtures', toSupport.fixtures);
```